src/controllers/grantAWSAccessCommand.ts

```diff
@@ -0,0 +1,32 @@
+import { discordTextResponse } from "../utils/discordResponse";
+import { SUPER_USER_ONE, SUPER_USER_TWO } from "../constants/variables";
+import { env } from "../typeDefinitions/default.types";
+import {
+  messageRequestMember,
+  messageRequestDataOptions,
+} from "../typeDefinitions/discordMessage.types";
+import { grantAWSAccess } from "../utils/awsAccess";
+
+export async function grantAWSAccessCommand(
+  transformedArgument: {
+    member: messageRequestMember;
+    userDetails: messageRequestDataOptions;
+    awsGroupDetails: messageRequestDataOptions;
+    channelId: number;
+  },
+  env: env,
+  ctx: ExecutionContext
+) {
+  const isUserSuperUser = [SUPER_USER_ONE, SUPER_USER_TWO].includes(
```

Comment:
Can we create this array in constants and import the array from there
This way if more superusers are added, we'll have to change in just one place

Reply:
super user Ids are in constants and we are importing them individually

Reply:
the constants thing it will be more flexible if the command fetches the super users according to its need rather than having a fixed set of super users